package.json

```diff
@@ -38,8 +42,16 @@
     "commitlint": "^8.3.5",
     "husky": "^4.2.5",
     "prettier": "^1.18.2",
-    "semantic-release": "^17.0.6"
+    "semantic-release": "^17.0.6",
+    "jasmine": "^3.5.0",
+    "@types/jasmine": "^3.5.10",
+    "atom-jasmine3-test-runner": "^4.4.4",
```

Comment:
What's the benefit of updating the test setup in this case? Bundled version is old, but we can check if package activates correctly with it, without extra dependencies.

Reply:
- Jasmine 3 is the version that is supported by the original project and community.
- Writing tests in Jasmine 1 is cumbersome and very clumsy. 
- Jasmine 3 has `await/async` which is not available in the old version. We don't need to care about the promise and other stuff anymore.
- Jasmine 3 has very good features such as `measure` that is used in this PR. https://github.com/UziTech/atom-jasmine3-test-runner
- There is no point in writing tests in the old version. Someone would do that if they have written the test years ago.